src/gpu/d3d12/SDL_gpu_d3d12.c

```diff
@@ -2473,12 +2473,31 @@ static D3D12GraphicsRootSignature *D3D12_INTERNAL_CreateGraphicsRootSignature(
     return d3d12GraphicsRootSignature;
 }
 
+static bool D3D12_INTERNAL_IsValidShaderBytecode(
+    const Uint8 *code,
```

Comment:
I'm not on a PC ATM, but is this variable null tested before it reaches this point?

Also applies to other instances.